client-js/app/scripts/rpc.ts

```diff
@@ -92,6 +92,7 @@ export class JsonRpcSocket {
    * Close the rpc socket by closing the underlying WebSocket.
    */
   close(code?: number, reason?: string) {
+    code = code || 1000;
```

Comment:
Passing a code of 1000 to the [close method](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket#close()) just means we're closing the socket normally because we're done with it, not because there was an error. There are lots of other [status codes](https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent#Status_codes), but typically we won't need them.